Homeworks/homework_16/base_suite.robot

```diff
@@ -0,0 +1,22 @@
+*** Settings ***
+Library           SeleniumLibrary
+Variables         variabels.py
+Resource          common_keywords.resource
+
+*** Test Cases ***
+User Registration and Login Test
+    Open Browser and Log In
+    [Teardown]    Close Browser
```

Comment:
Could be moved to suite Setting section as it's the same for both tests in the suite
```
*** Settings ***
Test Teardown    Close Browser
```

Reply:
Test Teardown    Close Browser has been added to Settings
Thank you!